config/settings-production.yml

```diff
@@ -42,13 +42,25 @@ buffy:
       remove_labels:
         - "0/pre-review-checks"
         - "New Submission!"
+      add_as_assignee: true
```

Comment:
This ensures that the editor is assigned to the issue.
I think it's ok if we have the EiC on the issue as well. that will help us track who's contributing to what packages. and right now we do not have a list of people who served as EiC on each review!